WebHostLib/static/styles/tooltip.css

```diff
@@ -57,12 +57,13 @@ give it one of the following classes: tooltip-left, tooltip-right, tooltip-top,
 .tooltip:after, [data-tooltip]:after {
 	z-index: 10000;
 	padding: 8px;
-	width: 160px;
+	width: 260px;
```

Comment:
```suggestion
	max-width: 260px;
```

Reply:
unfortunately removing width makes the ones on /generate really short and messy.